src/integrations/admin/admin-columns-cache-integration.php

```diff
@@ -109,7 +109,9 @@ public function fill_cache() {
 		$indexables = $this->indexable_repository->find_by_multiple_ids_and_type( $post_ids, 'post', false );
 
 		foreach ( $indexables as $indexable ) {
-			$this->indexable_cache[ $indexable->object_id ] = $indexable;
+			if ( $indexable !== false ) {
```

Comment:
What about this instead? We'd be sure it also has the expected property
```suggestion
			if ( $indexable instanceof Indexable ) {
```

Reply:
Makes total sense! This would be even more secure.